qualtran/bloqs/arithmetic.py

```diff
@@ -26,7 +26,6 @@
 
 if TYPE_CHECKING:
     from qualtran.resource_counting import SympySymbolAllocator
-    from qualtran.simulation.classical_sim import ClassicalValT
```

Comment:
please revert this change. this is indeed used in type annotations

Reply:
Done.

Reply:
this change is still part of the PR

Reply:
sorry. fixed.